### PR TITLE
docs(ja): translate `Philosophy` into Japanese 

### PR DESCRIPTION
**Pull request by yoshi2no**
## Summary

Partial work of https://github.com/biomejs/biome/issues/880. `internals/philosophy`

## Test Plan

- Check if all translations make sense

### CODE REVIEW

**Comment by nissy-dev** [comment]
I'm reviewing

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+このリストには、プロジェクトが守るべき一般的な倫理が含まれています。
+このリストは網羅的なものではありません。いくつかは明白なものですが、完全性のために記載されています。
+
+## プロジェクト管理
+
+- **明確な見通しを設定する。** プロジェクトの意図と決定事項を事前に十分に周知します。
+  予期せぬことは何も起こらないようにするべきです。
+
+- **決定事項を明確に伝える。** チームは、プライベートチャンネルを使って選択肢を評価し、決定を下すことがあります。
+  チームは[GitHub discussions](https://github.com/biomejs/biome/discussions)や[Discord](https://discord.gg/BypW39g6Yc)のようなパブリックチャンネルを使用して議論を行おうとしますが、民間企業の性質上、頻繁にプライベートなチェックインを行うのが一般的です。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:15]:
```suggestion
  チームは[GitHub discussions](https://github.com/biomejs/biome/discussions)や[Discord](https://discord.gg/BypW39g6Yc)のようなパブリックチャンネルを使用して議論を行おうとしますが、通常の企業ではしばしばプライベートなチャンネルで議論してしまいがちです。
```

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  プライベートチャンネルで決定が行われた場合、チームはこれらの決定をパブリックチャンネルを使用して伝えることを約束する必要があります。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:16]:
```suggestion
  プライベートチャンネルで決定が行われた場合、チームはこれらの決定をパブリックチャンネルを使用して伝える必要があります。
```

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## 技術的なこと
+
+- **エラーは、可能な限り修正の提案やヒントを示すべきです。** 無関係または役に立たないメッセージが表示されないように、これらは使用状況から推論・フィルタリングされるべきです。
+
+- **ユニークで具体的なエラーメッセージ。** 一般的なエラーメッセージは避けるべきです。
+  これにより、ユーザーは何が間違っていたのかを理解しやすくなり、メンテナーはユニークな呼び出し箇所とデバッグに必要な情報を得ることができます。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:23]:
```suggestion
  これにより、ユーザーは何が間違っていたのかを理解しやすくなり、メンテナーは具体的な呼び出し箇所とデバッグに必要な情報を得ることができます。
```

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- **明確な見通しを設定する。** プロジェクトの意図と決定事項を事前に十分に周知します。
+  予期せぬことは何も起こらないようにするべきです。
+
+- **決定事項を明確に伝える。** チームは、プライベートチャンネルを使って選択肢を評価し、決定を下すことがあります。
+  チームは[GitHub discussions](https://github.com/biomejs/biome/discussions)や[Discord](https://discord.gg/BypW39g6Yc)のようなパブリックチャンネルを使用して議論を行おうとしますが、民間企業の性質上、頻繁にプライベートなチェックインを行うのが一般的です。
+  プライベートチャンネルで決定が行われた場合、チームはこれらの決定をパブリックチャンネルを使用して伝えることを約束する必要があります。
+
+## 技術的なこと
+
+- **エラーは、可能な限り修正の提案やヒントを示すべきです。** 無関係または役に立たないメッセージが表示されないように、これらは使用状況から推論・フィルタリングされるべきです。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:20]:
```suggestion
- **エラーは、可能な限り修正の提案やヒントを示すべきです。** 無関係で役に立たないメッセージが表示されないように、使用状況によって補足やフィルタリングされるべきです。
```

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+- **ユニークで具体的なエラーメッセージ。** 一般的なエラーメッセージは避けるべきです。
+  これにより、ユーザーは何が間違っていたのかを理解しやすくなり、メンテナーはユニークな呼び出し箇所とデバッグに必要な情報を得ることができます。
+
+- **APIを最適化する。** すべてのオプションとフラグの必要性を問いかけてください。
+  それらは本当に必要ですか？ 組み合わせることはできませんか？ どうしたらコードの分岐を減らすことができますか？
+
+- **専門用語を控える。** ユーザーが特定の専門用語を理解しているとは考えないでください。
+  専門家と初心者の両方に正確な意味を提供するよう努めてください。例えば、パーサーエラーを出す際に"token"という用語の代わりに"character"を使用するなどです。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:29]:
```suggestion
  専門家と初心者の両方に正確な意味を提供するよう努めてください。例えば、パーサーのエラーを出す際に"token"という用語の代わりに"character"を使用するなどです。
```

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+- **コマンドやフラグの命名には冗長性を持たせる。** 不必要で紛らわしい略語は使用しないでください。
+
+- **包括的な用語を使用する。** 性別にとらわれない代名詞を使用してください。能力主義的な中傷をしたり無神経と思われる用語を使用しないでください。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:33]:
```suggestion
- **包括的な用語を使用する。** 性別にとらわれない代名詞を使用してください。障害者の差別や無神経と思われる用語を使用しないでください。
```

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+- **汎用的なクライアント向けに構築する。** ターミナルがANSIコードを使用した出力のみを処理するとは限らないということを前提にしてください。
+  IDE、ブラウザ、その他の環境での閲覧に適した抽象化を使用してください。
+
+- **ターミナルの出力は明確であるべきです。** ターミナルの出力を設計する際には、色のようなフォーマットの手掛かりだけに頼らないでください。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:38]:
```suggestion
- **ターミナルの出力は明確であるべきです。** ターミナルの出力を設計する際には、色のような書式だけに頼らないでください。
```

File: website/src/content/docs/ja/internals/philosophy.mdx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+  常にフォーマット、記号、スペーシングの組み合わせを使用してください。もし全てのANSIコードが取り除かれても、全ての出力が理解できるようにしてください。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/philosophy.mdx:39]:
```suggestion
  常に書式、記号、スペーシングの組み合わせを使用してください。もし全てのANSIコードが取り除かれても、全ての出力が理解できるようにしてください。
```

**Review by nissy-dev — CHANGES_REQUESTED** [review]
Thank you for the contribution. I left some suggestions.

**Comment by yoshi2no** [comment]
@nissy-dev 
Thank you for your review! I have fixed all of them!

**Review by nissy-dev — APPROVED** [review]

**Pull request merged**